webui/webpack.config.js

```diff
@@ -83,7 +83,7 @@ module.exports = {
         test: /\.elm$/,
         exclude: [/elm-stuff/, /node-modules/],
         use: [
-          { loader: 'elm-hot-loader' },
+          // { loader: 'elm-hot-loader' },
```

Comment:
Why did you comment out the hot loader?

Reply:
It broke with some elm module that I introduced.  I'm not sure why, but I wanted to take a look first.